cellprofiler/modules/blobdetection.py

```diff
@@ -0,0 +1,379 @@
+# coding=utf-8
+
+"""
+
+Find blobs in a grayscale (single-channel) image or volume. Blobs are bright on dark or dark on bright regions in an
+image. The output of this module is a binary image of white circles or spheres centered around found blobs on a black
```

Comment:
Nice. 😎

I assumed it’d return boundaries, but filled circles are probably more useful.